### ISSUE TITLE
Vite Failed to resolve entry for package "https"

### ISSUE THREAD

**Issue by DuskXi** [freddy38510/quasar-app-extension-ssg#295]
## The package may have incorrect main/module/exports specified in its package.json: Failed to resolve entry for package "https".
When I try to execute quasar ssg generate on my quasar vite project, I encounter the following error:
```console
PS D:\projects\ArkRecord-Web> quasar ssg generate
 App • Looking for Quasar App Extension "ssg" command "generate"
 App • Running "ssg" Quasar App Extension...
 App • Running "ssg" > "generate" command

 Build mode............. ssg
 Pkg quasar............. v2.7.7
 Pkg @quasar/app-vite... v1.0.6
 Pkg vite............... v2.9.15
 Pkg ssg................ v4.5.7
 Debugging.............. no

 App • Cleaned build artifact: "D:\projects\ArkRecord-Web\dist\ssg"
 App • Cleaned build artifact: "D:\projects\ArkRecord-Web\node_modules\.cache\quasar-app-extension-ssg"
 App •  WAIT  • Compiling of SSR Client with Vite in progress...
[vite:resolve] Failed to resolve entry for package "https". The package may have incorrect main/module/exports specified in its package.json: Failed to resolve entry for package "https". The package may have incorrect main/module/exports specified in its package.json.
D:\projects\ArkRecord-Web\node_modules\vite\dist\node\chunks\dep-689425f3.js:40970
    throw new Error(`Failed to resolve entry for package "${id}". ` +
          ^

Error: Failed to resolve entry for package "https". The package may have incorrect main/module/exports specified in its package.json: Failed to resolve entry for package "https". The package may have incorrect main/module/exports specified in its package.json.
    at packageEntryFailure (D:\projects\ArkRecord-Web\node_modules\vite\dist\node\chunks\dep-689425f3.js:40970:11)
    at resolvePackageEntry (D:\projects\ArkRecord-Web\node_modules\vite\dist\node\chunks\dep-689425f3.js:40966:9)
    at tryNodeResolve (D:\projects\ArkRecord-Web\node_modules\vite\dist\node\chunks\dep-689425f3.js:40773:20)
    at Object.resolveId (D:\projects\ArkRecord-Web\node_modules\vite\dist\node\chunks\dep-689425f3.js:40581:28)
    at D:\projects\ArkRecord-Web\node_modules\vite\node_modules\rollup\dist\shared\rollup.js:22826:37 {
  code: 'PLUGIN_ERROR',
  plugin: 'vite:resolve',
  hook: 'resolveId',
  watchFiles: [
    'D:/projects/ArkRecord-Web/index.html',
    'D:/projects/ArkRecord-Web/.quasar/client-entry.js',
    'D:\\projects\\ArkRecord-Web\\node_modules\\vite\\package.json',
    'D:\\projects\\ArkRecord-Web\\node_modules\\vue\\package.json',
    'D:\\projects\\ArkRecord-Web\\node_modules\\@quasar\\extras\\package.json',
    'D:\\projects\\ArkRecord-Web\\node_modules\\quasar\\package.json',
    'D:\\projects\\ArkRecord-Web\\node_modules\\quasar-app-extension-ssg\\roboto-font\\roboto-font.css',
    'D:\\projects\\ArkRecord-Web\\node_modules\\quasar-app-extension-ssg\\package.json',
    'D:/projects/ArkRecord-Web/node_modules/vue/dist/vue.runtime.esm-bundler.js',
    'D:/projects/ArkRecord-Web/node_modules/@quasar/extras/material-icons/material-icons.css',
    'D:/projects/ArkRecord-Web/node_modules/quasar/dist/quasar.sass',
    'D:/projects/ArkRecord-Web/.quasar/app.js',
    'D:/projects/ArkRecord-Web/.quasar/quasar-user-options.js',
    'D:/projects/ArkRecord-Web/node_modules/quasar-app-extension-ssg/src/boot/ssg-corrections.js',
    'D:/projects/ArkRecord-Web/src/css/app.scss',
    'vite/preload-helper',
    'D:\\projects\\ArkRecord-Web\\node_modules\\@vue\\runtime-dom\\package.json',
    'D:/projects/ArkRecord-Web/node_modules/@vue/runtime-dom/dist/runtime-dom.esm-bundler.js',
    'D:/projects/ArkRecord-Web/node_modules/quasar/src/plugins/Platform.js',
    'D:/projects/ArkRecord-Web/node_modules/quasar/src/vue-plugin.js',
    'D:/projects/ArkRecord-Web/src/App.vue',
    'D:/projects/ArkRecord-Web/src/router/index.js',
    'D:/projects/ArkRecord-Web/node_modules/quasar/lang/zh-CN.mjs',
    'D:/projects/ArkRecord-Web/node_modules/quasar/src/plugins/Notify.js',
    'D:/projects/ArkRecord-Web/node_modules/quasar/src/plugins/Dialog.js',
    'D:/projects/ArkRecord-Web/node_modules/quasar/src/plugins/Loading.js',
    'plugin-vue:export-helper',
    'D:\\projects\\ArkRecord-Web\\node_modules\\@vue\\runtime-core\\package.json',
    'D:\\projects\\ArkRecord-Web\\node_modules\\@vue\\shared\\package.json',
    'D:/projects/ArkRecord-Web/node_modules/@vue/runtime-core/dist/runtime-core.esm-bundler.js',
    'D:/projects/ArkRecord-Web/node_modules/@vue/shared/dist/shared.esm-bundler.js',
    'D:/projects/ArkRecord-Web/node_modules/quasar/src/install-quasar.js',
    'D:/projects/ArkRecord-Web/node_modules/quasar/src/lang.js',
    'D:/projects/ArkRecord-Web/node_modules/quasar/src/icon-set.js',
    'D:\\projects\\ArkRecord-Web\\node_modules\\vue-router\\package.json',
    'D:/projects/ArkRecord-Web/node_modules/quasar/wrappers/index.mjs',
    'D:/projects/ArkRecord-Web/node_modules/vue-router/dist/vue-router.esm-bundler.js',
    'D:/projects/ArkRecord-Web/src/router/routes.js',
    'D:/projects/ArkRecord-Web/node_modules/quasar/src/utils/private/inject-obj-prop.js',
    'D:/projects/ArkRecord-Web/node_modules/quasar/src/components/avatar/QAvatar.js',
    'D:/projects/ArkRecord-Web/node_modules/quasar/src/components/icon/QIcon.js',
    'D:/projects/ArkRecord-Web/node_modules/quasar/src/components/btn/QBtn.js',
    'D:/projects/ArkRecord-Web/node_modules/quasar/src/components/spinner/QSpinner.js',
    'D:/projects/ArkRecord-Web/node_modules/quasar/src/utils/private/create.js',
    'D:/projects/ArkRecord-Web/node_modules/quasar/src/utils/event.js',
    'D:/projects/ArkRecord-Web/node_modules/quasar/src/utils/private/global-nodes.js',
    'D:/projects/ArkRecord-Web/node_modules/quasar/src/utils/private/is.js',
    'D:/projects/ArkRecord-Web/node_modules/quasar/src/components/dialog-plugin/DialogPlugin.js',
    'D:/projects/ArkRecord-Web/node_modules/quasar/src/utils/private/global-dialog.js',
    'D:/projects/ArkRecord-Web/node_modules/quasar/src/utils/private/define-reactive-plugin.js',
    'D:/projects/ArkRecord-Web/node_modules/quasar/src/utils/prevent-scroll.js',
    'D:/projects/ArkRecord-Web/node_modules/quasar/src/plugins/Screen.js',
    'D:/projects/ArkRecord-Web/node_modules/quasar/src/plugins/Dark.js',
    'D:/projects/ArkRecord-Web/node_modules/quasar/src/history.js',
    'D:/projects/ArkRecord-Web/node_modules/quasar/src/body.js',
    'D:/projects/ArkRecord-Web/node_modules/quasar/src/utils/private/symbols.js',
    'D:/projects/ArkRecord-Web/node_modules/quasar/src/utils/private/global-config.js',
    'D:/projects/ArkRecord-Web/node_modules/quasar/lang/en-US.mjs',
    'D:/projects/ArkRecord-Web/node_modules/quasar/icon-set/material-icons.mjs',
    'D:/projects/ArkRecord-Web/src/layouts/MainLayout.vue',
    'D:/projects/ArkRecord-Web/src/pages/IndexPage.vue',
    'D:/projects/ArkRecord-Web/src/layouts/PopupLayout.vue',
    'D:/projects/ArkRecord-Web/src/pages/Popup.vue',
    'D:/projects/ArkRecord-Web/src/pages/IndexPage_backup.vue',
    'D:/projects/ArkRecord-Web/src/pages/DebugPages.vue',
    'D:/projects/ArkRecord-Web/src/pages/ErrorNotFound.vue',
    'D:/projects/ArkRecord-Web/node_modules/quasar/src/composables/private/use-size.js',
    'D:/projects/ArkRecord-Web/node_modules/quasar/src/utils/private/render.js',
    'D:/projects/ArkRecord-Web/node_modules/quasar/src/directives/Ripple.js',
    'D:/projects/ArkRecord-Web/node_modules/quasar/src/components/btn/use-btn.js',
    'D:/projects/ArkRecord-Web/node_modules/quasar/src/utils/private/key-composition.js',
    'D:/projects/ArkRecord-Web/node_modules/quasar/src/components/spinner/use-spinner.js',
    'D:\\projects\\ArkRecord-Web\\node_modules\\@vue\\devtools-api\\package.json',
    'D:/projects/ArkRecord-Web/node_modules/@vue/devtools-api/lib/esm/index.js',
    'D:/projects/ArkRecord-Web/node_modules/quasar/src/components/dialog/QDialog.js',
    'D:/projects/ArkRecord-Web/node_modules/quasar/src/components/card/QCard.js',
    'D:/projects/ArkRecord-Web/node_modules/quasar/src/components/card/QCardSection.js',
    'D:/projects/ArkRecord-Web/node_modules/quasar/src/components/card/QCardActions.js',
    'D:/projects/ArkRecord-Web/node_modules/quasar/src/components/separator/QSeparator.js',
    'D:/projects/ArkRecord-Web/node_modules/quasar/src/components/input/QInput.js',
    'D:/projects/ArkRecord-Web/node_modules/quasar/src/components/option-group/QOptionGroup.js',
    'D:/projects/ArkRecord-Web/node_modules/quasar/src/composables/private/use-dark.js',
    'D:\\projects\\ArkRecord-Web\\node_modules\\@vue\\reactivity\\package.json',
    'D:/projects/ArkRecord-Web/node_modules/@vue/reactivity/dist/reactivity.esm-bundler.js',
    'D:/projects/ArkRecord-Web/node_modules/quasar/src/utils/scroll.js',
    'D:/projects/ArkRecord-Web/node_modules/quasar/src/utils/debounce.js',
    'D:/projects/ArkRecord-Web/node_modules/quasar/src/utils/set-css-var.js',
    'D:/projects/ArkRecord-Web/node_modules/quasar/src/components/page/QPageContainer.js',
    'D:/projects/ArkRecord-Web/node_modules/quasar/src/components/layout/QLayout.js',
    'D:/projects/ArkRecord-Web/src/layouts/MainLayout.vue?vue&type=style&index=0&lang.css',
    'D:/projects/ArkRecord-Web/src/utils/largeFileStorage.js',
    'D:/projects/ArkRecord-Web/src/utils/storage.js',
    'D:/projects/ArkRecord-Web/node_modules/quasar/src/components/badge/QBadge.js',
    'D:/projects/ArkRecord-Web/node_modules/quasar/src/components/toggle/QToggle.js',
    'D:/projects/ArkRecord-Web/node_modules/quasar/src/components/btn-toggle/QBtnToggle.js',
    'D:/projects/ArkRecord-Web/node_modules/quasar/src/components/page-scroller/QPageScroller.js',
    'D:/projects/ArkRecord-Web/node_modules/quasar/src/components/page/QPage.js',
    'D:/projects/ArkRecord-Web/package.json',
    'D:/projects/ArkRecord-Web/src/utils/data.js',
    'D:/projects/ArkRecord-Web/src/components/tabs/ChartShow.vue',
    ... 115 more items
  ]
}

```

I can't any useful information from google searching, so I can here to send this issue.

**Comment by freddy38510** [comment]
Hello @DuskXi ,

It should be fixed in the [latest release v4.5.8](https://www.npmjs.com/package/quasar-app-extension-ssg/v/4.5.8).

Could you upgrade the extension and confirm it please ?

**Comment by DuskXi** [comment]
It is working now, thank you.
